logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -33,22 +29,27 @@ class LogitechR400Usb(object):
 
     from logitech_r400 import LogitechR400Usb
     with LogitechR400Usb() as pointer:
-        pointer.read_data()
+        pointer._read_data()
     """
 
     __interface = 0
 
     def __init__(self):
-        self._devices = self.find_devices()
         self._state = []
         self._pubs = []
+        self.product = None
```

Comment:
Product and vendor are private members, so they should have a leading underscore (like the ones above).